README.md

```diff
@@ -1,7 +1,7 @@
 # GoAws
 [![Build Status](https://travis-ci.org/p4tin/goaws.svg?branch=master)](https://travis-ci.org/p4tin/goaws)
 
-You are always welcome to [tweet me](https://twitter.com/gocodecloud) or [buy me a coffee](https://www.paypal.me/p4tin)
+You are always welcome to [tweet the creator in chief](https://twitter.com/gocodecloud) or [buy him a coffee](https://www.paypal.me/p4tin)
```

Comment:
You can just remove that or put your name there - lol

Reply:
Hahaha I'll leave it up for you for a bit - you have been doing the hard work here for a long and I'm not set up!